src/redux/books/bookSlice.js

```diff
@@ -1,27 +1,45 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const id = 'OQo0tm9K5JZRXbAJo9Th';
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
 
 const initialState = {
-  books: [
-    {
-      item_id: 'item1',
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      item_id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
+  books: [],
+};
+
+export const fetchBooks = createAsyncThunk('book/fetchBooks', async () => {
+  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OQo0tm9K5JZRXbAJo9Th/books');
+  const data = await response.json();
+  return data;
+});
+
+export const postABook = createAsyncThunk('book/addBook', async (book) => {
```

Comment:
- [ ] @Ashnaali3255 As the previous reviewer noted  when you add a book it only adds to the UI but once the page is refreshed it disappears, this is because you are using `addBook` function instead of `postABook` which will store your data in the api, you can also integrate your postABook method in your addBook